### PR TITLE
CI: Pin .NET SDK 6.0.102 for now to avoid dotnet format regression

### PR DESCRIPTION
**Pull request by mfkl**
Fixes https://github.com/G-Research/consuldotnet/runs/5330947266?check_suite_focus=true

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup .NET SDK v6.0.x
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 6.0.102
```
Comment (marcin-krystianc) [inline .github/workflows/ci.yml:22]: Can you add a comment why we are pinning dotnet version ? You can also include a link to dotnet issue.

**Comment by mfkl** [comment]
It's all in the commit message, but I can add a comment in the code as well
if you prefer.

**Comment by marcin-krystianc** [comment]
> It's all in the commit message, but I can add a comment in the code as well if you prefer.

Yeah, I think it is good to include such information in the code as well. So next person who sees it knows the reason for it without using extra tool like `git blame`.

**Comment by mfkl** [comment]
Ok

**Review by marcin-krystianc — APPROVED** [review]

**Review by jgiannuzzi — APPROVED** [review]

**Pull request merged**